src/tools/measure.py

```diff
@@ -1,10 +1,22 @@
-from numpy import arange, random, log2
+from numpy import arange, random, log2, ndarray
+from numpy.typing import NDArray
+from typing import Union
 from .probability import probability
 from .base import convert_state
+from ..errors import *
+from ..quantum_circuit import QuantumCircuit
 
 
-def measure(state):
-    state = convert_state(state)
+def measure(quantumstate: Union[QuantumCircuit, ndarray]) -> NDArray:
+    """Outputs the measure of a quantum circuit state.
+    ```
+    from qcpy import quantumcircuit, measure
+    measure(quantumcircuit(qubits = 2)
```

Comment:
I am not trying to nitpick but I realized there is a missing parenthesis here lol

Reply:
Thanks for being picky fixed it